sdk/messaging/azservicebus/admin/main_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package admin
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/joho/godotenv"
+)
+
+func TestMain(m *testing.M) {
+	err := godotenv.Load("../.env")
+
+	if err != nil {
+		log.Printf("Failed to load env file, no live tests will run: %s", err.Error())
```

Comment:
Will this work in our DevOps pipeline?

Reply:
It just errors out harmlessly in DevOps.

Reply:
This is a "local dev" affordance.

Reply:
I mean, do the live tests run in DevOps?

Reply:
Oh, yes they do.